vector-app/src/debug/java/im/vector/app/features/debug/features/DebugFeaturesStateFactory.kt

```diff
@@ -100,11 +100,6 @@ class DebugFeaturesStateFactory @Inject constructor(
                                 key = DebugFeatureKeys.reciprocateQrCodeLogin,
                                 factory = VectorFeatures::isReciprocateQrCodeLogin
                         ),
-                        createBooleanFeature(
-                                label = "Enable Voice Broadcast",
-                                key = DebugFeatureKeys.voiceBroadcastEnabled,
-                                factory = VectorFeatures::isVoiceBroadcastEnabled
-                        ),
```

Comment:
Adding a lab setting should not make us to remove the feature flag. The feature flag could help to control the display of the lab setting. Not blocking this PR though.